fgrpc/dapr.go

```diff
@@ -153,6 +170,37 @@ func (d *DaprGRPCRunnerResults) prepareRequest4PubSub(o *GRPCRunnerOptions) erro
 		} else {
 			d.publishEventRequest.Data = []byte{}
 		}
+	case "publish-multi":
+		d.publishEventRequests = make([]*dapr.PublishEventRequest, numEventsInt)
+		for i := 0; i < numEventsInt; i++ {
+			d.publishEventRequests[i] = &dapr.PublishEventRequest{
+				PubsubName:      store,
```

Comment:
Should the name be something else than `store` so that no confusion arises, while later analyzing logs etc.

Reply:
It comes from `DaprRequestParameters`, I would track these changes separately.